projects/docs-tools/src/modules/type-definitions/typedoc-adapter.service.spec.ts

```diff
@@ -281,15 +314,7 @@ describe('TypeDoc adapter', () => {
           setSignature: [
             {
               name: '__set',
-              comment: {
-                shortText: 'The foo of the FooClass.',
```

Comment:
Possible to keep existing tests as-is so we're sure your changes don't negatively affect anything else? Maybe add another, separate accessor that will test your changes?

Reply:
The issue here was that the previous test actually exhibited one of the bugs here. In certain situations there is only a setter and the comment data needs to come from there. The previous test was saying this should be ignored -but that isn't true. My new tests (now 2 instead of the original 1) are now more accurate to what should be happening.